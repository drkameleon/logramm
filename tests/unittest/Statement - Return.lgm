try(x) =>
{
	out "Before";

	i = 0;
	@ i<x :
	{
		out "BEFORE : i = " + i;
		i==4 :
			x==5 : ==> x*2;
		i==4 :==> x*3;
		out "AFTER : i = " + i;
		i += 1;
	}
	out "After";
	out "After";
	out "After";
	out "After";
}

out try(5);