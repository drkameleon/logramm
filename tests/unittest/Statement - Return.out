Before
BEFORE : i = 0
AFTER : i = 0
BEFORE : i = 1
AFTER : i = 1
BEFORE : i = 2
AFTER : i = 2
BEFORE : i = 3
AFTER : i = 3
BEFORE : i = 4
10
